appendix.rst

```diff
@@ -45,9 +45,6 @@ below with their respective functionality.
 #. **SINGULARITY_BOOT**: Set to false by default, considers if executing
    ``/sbin/init`` when container boots (root only).
 
-#. **SINGULARITY_BUILDER**: To specify the remote builder service URL.
```

Comment:
https://github.com/apptainer/apptainer-userdocs/pull/32/files#diff-c0d464e2c03157a8a8c5783f7f354d6dae4579e5de53cc337ee5c50ce319c30aL257 this environment variable should also be removed because it indicates that a remote build should take place.

Reply:
**SINGULARITY_BUILDER**: is already removed.